### ISSUE TITLE
[A11y] Work Show Page-All - Keyboard navigation

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1993]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j
https://staging.oregondigital.org/concern/documents/df727057n
https://staging.oregondigital.org/concern/images/1g05fb620

- When tabbing to the universal viewer, a hidden iframe gets focus. I don't know what this iframe is supposed to do, but it makes keyboard focus just disappear.
- Somewhere within that same iframe there's a link to the homepage. You can activate it and get the homepage to display within the iframe. Obviously this is bad for usability, but since the link is only available to keyboard users, it's especially bad for anybody navigating by keyboard alone. This link doesn't have any kind of accessible name or label, so it's extra confusing because a screen reader doesn't even announce that you're on a link.

### Suggested Fix

I can't tell if the iframe is containing something and the DOM view is just "wrong", or if it's truly an empty iframe. If empty, remove it entirely. If it's necessary, it at least needs a title. I'm not sure how to give better advice here, because Accessibility Insights doesn't see this iframe even though I can find it in the page source and DOM, and I can see it "capturing" the focus.

The weird link to the homepage is similar - it seems to somehow be within the iframe. I'm not sure how to identify what's going on because I can't find that link in the DOM, but it needs to go away....

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Depends on fix
